test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1492,6 +1493,76 @@ public void testEnforceSnapshotSync() throws Exception {
         assertThat(oldShadowTs).isEqualTo(Long.parseLong(metadata.getVal()));
     }
 
+    @Test
+    public void testSnapshotApplyInterrupted() throws Exception {
+        // Write first batch of entries to active map
+        for (int i = 0; i < firstBatch; i++) {
+            try (TxnContext txn = activeCorfuStore.txn(NAMESPACE)) {
+                txn.putRecord(mapActive, StringKey.newBuilder().setKey(String.valueOf(i)).build(),
+                        IntValue.newBuilder().setValue(i).build(), null);
+                txn.commit();
+            }
+        }
+        assertThat(mapActive.count()).isEqualTo(firstBatch);
+        assertThat(mapStandby.count()).isZero();
+
+
+        int waitInSnapshotApplyMs = 10000;
+        activeReplicationServer = runReplicationServer(activeReplicationServerPort);
+        standbyReplicationServer = runReplicationServer(standbyReplicationServerPort, waitInSnapshotApplyMs);
```

Comment:
This actually tests another flow where the apply is yet to happen on the sink.
The fix would not be exercised..

I think we should use the framework in LogReplicationIT. It gives us a way to simulate loss of msg packets from the sink.

Reply:
Reordered the test operations so we hit the right case, can see this log from the fix now:

```
2025-07-09T22:32:10.993Z | INFO  |                        netty-0 | .l.t.s.CorfuNettyClientChannel | Received msg type=LR_METADATA_RESPONSE
2025-07-09T22:32:11.012Z | INFO  |             runtime-fsm-worker | .LogReplicationMetadataManager | Replication status snapshot sync info for baseSnapshot 25 is being updated from ONGOING to COMPLETED
2025-07-09T22:32:11.029Z | INFO  |             runtime-fsm-worker | o.c.i.l.r.fsm.NegotiatingState | Resume LOG ENTRY sync. Address space has not been trimmed, deltas are guaranteed to be available. logHead=0, lastLogProcessed=25
```